tests/constraints/tests.py

```diff
@@ -80,21 +83,38 @@ def test_repr(self):
     def test_deconstruction(self):
         fields = ['foo', 'bar']
         name = 'unique_fields'
-        check = models.UniqueConstraint(fields=fields, name=name)
-        path, args, kwargs = check.deconstruct()
+        constraint = models.UniqueConstraint(fields=fields, name=name)
```

Comment:
Would be better to do unrelated cleanups like this (I saw another rename in schema.py) in another commit.

Reply:
removed, should I add new commit in this MR or other?

Reply:
sure, you can add it here.

Reply:
added as new commit